client/src/main/scala/drt/client/modules/GoogleAnalytics.scala

```diff
@@ -1,26 +1,82 @@
 package drt.client.modules
 
-
+import org.scalajs.dom
 import scala.scalajs.js
 import scala.scalajs.js.annotation.JSGlobalScope
 
 object GoogleEventTracker {
-  def isScriptLoaded: Boolean =js.Dynamic.global.analytics.isInstanceOf[js.Function]
-  def sendPageView(page:String):Unit={
-    if (isScriptLoaded) GoogleAnalytics.analytics("send","event","pageView", page)
+  val trackingCode: GoogleTrackingCode = GoogleTrackingCode.getTrackingCode(dom.document.location.href)
+  def port: String = dom.document.getElementById("port-code").getAttribute("value")
+  def userId: String = dom.document.getElementById("user-id").getAttribute("value")
```

Comment:
We can't store names and emails, but we can store UUID's. I thought this was a UUID, isn't it?

Reply:
yes, good point!

Reply:
If there are some User Ids which are email addresses, I could just not track them, or encode them somehow so that they are not readable. So you think that is that something we should do?